wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/DCMotorSim.java

```diff
@@ -51,6 +52,36 @@ public DCMotorSim(
     m_gearing = gearing;
   }
 
+  /**
+   * Creates a simulated DC motor mechanism.
+   *
+   * @param kV The velocity gain, in volts/(unit/sec)
+   * @param kA The acceleration gain, in volts/(unit/sec²)
+   * @param gearbox The type of and number of motors in the DC motor gearbox.
+   * @param gearing The gearing of the DC motor (numbers greater than 1 represent reductions).
+   */
+  public DCMotorSim(double kV, double kA, DCMotor gearbox, double gearing) {
+    super(identifySystem(kV, kA));
+    m_gearbox = gearbox;
+    m_gearing = gearing;
+  }
+
+  /**
+   * Creates a simulated DC motor mechanism.
+   *
+   * @param kV The velocity gain, in volts/(unit/sec)
+   * @param kA The acceleration gain, in volts/(unit/sec²)
+   * @param gearbox The type of and number of motors in the DC motor gearbox.
+   * @param gearing The gearing of the DC motor (numbers greater than 1 represent reductions).
+   * @param measurementStdDevs The standard deviations of the measurements.
+   */
+  public DCMotorSim(
+      double kV, double kA, DCMotor gearbox, double gearing, Matrix<N2, N1> measurementStdDevs) {
```

Comment:
Are you sure the gearbox and gearing are still needed? Kv and Ka already inherently include them.

Reply:
I assumed they were still needed since the existing plant constructor still included them for calculating current draw. The extra parameters definitely make it more clunky to use though.

Reply:
It's probably OK as long as we document what they're used for internally.